https://github.com/resurrecting-open-source-projects/scrot/pull/175

From: Sam James <sam@gentoo.org>
Date: Sun, 19 Dec 2021 23:59:45 +0000
Subject: [PATCH 1/2] build: respect docdir

Before this patch with --docdir=/usr/share/doc/scrot-1.7-r1 (example downstream
versioning for patches):
```
>>> /usr/share/doc/scrot-1.7-r1/
>>> /usr/share/doc/scrot-1.7-r1/README.md.bz2
>>> /usr/share/doc/scrot-1.7-r1/ChangeLog.bz2
>>> /usr/share/doc/scrot-1.7-r1/AUTHORS.bz2
>>> /usr/share/doc/scrot/
>>> /usr/share/doc/scrot/scrot.png
>>> /usr/share/doc/scrot/README.md.bz2
>>> /usr/share/doc/scrot/ChangeLog.bz2
>>> /usr/share/doc/scrot/AUTHORS.bz2
```

After:
```
>>> /usr/share/doc/scrot-1.7-r1/scrot.png
>>> /usr/share/doc/scrot-1.7-r1/README.md.bz2
>>> /usr/share/doc/scrot-1.7-r1/ChangeLog.bz2
>>> /usr/share/doc/scrot-1.7-r1/AUTHORS.bz2
```

This avoids installing duplicate files & installs them to the
same location but it's overridable by --docdir in ./configure which
helps downstreams.

Bug: https://bugs.gentoo.org/828928
Signed-off-by: Sam James <sam@gentoo.org>
--- a/Makefile.am
+++ b/Makefile.am
@@ -34,10 +34,9 @@ MAINTAINERCLEANFILES = Makefile.in
 
 man_MANS = man/scrot.1
 
-docs_DATA = README.md AUTHORS ChangeLog doc/scrot.png
-docsdir = $(prefix)/share/doc/scrot
+doc_DATA = README.md AUTHORS ChangeLog doc/scrot.png
 
-EXTRA_DIST = man/scrot.1 $(docs_DATA)
+EXTRA_DIST = man/scrot.1
 
 SUBDIRS = src
 
From: Sam James <sam@gentoo.org>
Date: Mon, 20 Dec 2021 00:03:35 +0000
Subject: [PATCH 2/2] build: simplify man page installation

Signed-off-by: Sam James <sam@gentoo.org>
--- a/Makefile.am
+++ b/Makefile.am
@@ -36,7 +36,7 @@ man_MANS = man/scrot.1
 
 doc_DATA = README.md AUTHORS ChangeLog doc/scrot.png
 
-EXTRA_DIST = man/scrot.1
+EXTRA_DIST = $(man_MANS)
 
 SUBDIRS = src
 
